usaspending_api/awards/v2/filters/matview_filters.py

```diff
@@ -222,9 +222,9 @@ def keyword_parse(keyword):
 
         elif key == "place_of_performance_scope":
```

Comment:
Was there any discussion of altering the `recipient_scope` filter to match?

Reply:
It wasn't discussed as part of this ticket. I can raise it as a question as to whether it is also an issue with the POs though, and see if we should make an additional ticket.